### PR TITLE
Start and stop integrated TURN servers inside ICE Plugin

### PR DESCRIPTION
**Pull request by FelonEkonom**

### CODE REVIEW

File: lib/membrane_ice_plugin/ice_bin.ex [hunk 1]
```diff
@@ -133,13 +133,18 @@ defmodule Membrane.ICE.Bin do
...
       controlling_mode: controlling_mode,
       port_range: port_range,
       handshake_module: hsk_module,
       handshake_opts: hsk_opts
     } = options
 
+    integrated_turn_servers =
+      if use_integrated_turn,
+        do: start_integrated_turn_servers(integrated_turn_ip),
+        else: []
```
Comment (mickel8) [inline lib/membrane_ice_plugin/ice_bin.ex:146]: I would move this code to `connector.ex` . As we discussed we can also get rid of `peer_pid` message.

File: lib/membrane_ice_plugin/ice_bin.ex [hunk 2]
```diff
@@ -108,10 +108,10 @@ defmodule Membrane.ICE.Bin do
...
                 description: "Indicator, if use integrated TURN"
               ],
-              integrated_turns_pids: [
-                spec: [pid()],
-                default: [],
-                description: "Pids of running integrated TURN servers"
+              integrated_turn_ip: [
+                spec: :inet.ip4_address() | nil,
+                default: nil,
+                description: "Address, where integrated TURN server will be set up"
```
Comment (mickel8) [inline lib/membrane_ice_plugin/ice_bin.ex:114]: `Address integrated TURN server will listen at`

**Review by mickel8 — CHANGES_REQUESTED** [review]

File: lib/membrane_ice_plugin/ice_bin.ex (hunk 1 above)
Comment (FelonEkonom) [reply lib/membrane_ice_plugin/ice_bin.ex]: Starting integrated TURN servers in `connector.ex` would make us have to send an extra message to `ICE.Bin` containing value holded in `integrated_turn_servers` variable, so I think that starting these TURN servers here is better solution

File: mix.exs [hunk 3]
```diff
@@ -44,7 +44,8 @@ defmodule Membrane.ICE.Plugin.Mixfile do
       {:dialyxir, "~> 1.0.0", only: :dev, runtime: false},
       {:credo, "~> 1.5", only: :dev, runtime: false},
       {:membrane_file_plugin, "~> 0.6.0", only: :test},
-      {:membrane_hackney_plugin, "~> 0.5.0", only: :test}
+      {:membrane_hackney_plugin, "~> 0.5.0", only: :test},
+      {:stun, github: "membraneframework/stun", branch: "removing-libnice"}
```
Comment (mickel8) [inline mix.exs:48]: Remember to use master before merging

**Review by mickel8 — APPROVED** [review]

**Pull request merged**